lib/util.ex

```diff
@@ -75,24 +70,51 @@ defmodule Util do
 
     case HTTPoison.post("https://api.start.gg/gql/alpha", JSON.encode!(body), headers) do
       {:ok, resp} ->
-        resp.body
-        |> JSON.decode
-        |> format_decoded_response
+        tourney_map = resp.body
+                      |> JSON.decode
```

Comment:
This nitpick is so small that I wouldn't bother including it if there weren't other comments to be fixed elsewhere, but: I very slightly prefer to include the parentheses to make it more obvious that a function is being called here.